apps/coscrad-frontend-e2e/src/e2e/alphabet/alphabet.e2e.cy.ts

```diff
@@ -0,0 +1,1015 @@
+const firstWord = 'apple';
+
+const secondWord = 'banana';
+
+const lastWord = 'zebra';
+
+const fifthWord = 'ear';
+
+const testAlphabetCharts = [
```

Comment:
Let's add a story to move this fixture data elsewhere. The tricky thing is that we'll want to be able to override the properties that we care about for this test (i.e., `firstWord`, etc.).